### PR TITLE
Add Onboarding card

### PR DESCRIPTION
**Pull request by ckopecky**
@bdougie 

Screen Shot 2019-11-09 at 8 03 19 PM

Screen Shot 2019-11-09 at 8 04 01 PM

### CODE REVIEW

File: stories/2-Card.stories.js [hunk 1]
```diff
@@ -130,3 +133,50 @@ export const NoteCard = () => (
...
+        <DoneChecking style={{width: "80%"}} src={done_checking}/>
+      </HintStyle>
+      <HintStyle style={{border: "none", minWidth: "15.5%", width: "15.5%"}}>
+        <PrimaryWithText />
+      </HintStyle>
+    </React.Fragment>
+  </Background>
+);
+
+export const OnboardingCard = () => (
```
Comment (bdougie) [inline stories/2-Card.stories.js]: apologies, I named this card screenshot wrong in the issue. Can  you rename to "PrimaryWithTextCard"?

**Comment by bdougie** [comment]
Everything looks good, just one small name change.

File: stories/2-Card.stories.js (hunk 1 above)
Comment (ckopecky) [reply stories/2-Card.stories.js]: You got it.

**Comment by ckopecky** [comment]
@bdougie ready 👍

**Pull request merged**